src/components/ScholarshipsMadeSimpleContent.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+import logopic from '../img/blank.png';
+import LoginButton from './LoginButton';
+
+function ContentText(tab) {
+  return (
+    <>
+      <Typography variant="overline" component="h6" gutterBottom>
+        {tab === 'students' ? 'students' : 'community contributor'}
+      </Typography>
+      <Typography variant="h2" component="h2" gutterBottom>
+        {tab === 'students'
+          ? 'Stress-free scholarships'
+          : 'Join and Support the Community'}
+      </Typography>
+      <Typography variant="body2" gutterBottom>
+        Et has minim eltir intellegat. Mea aeterno elefiend antiopam ad, nam no
+        suscipit quaerendum. At name mininum ponderum. Est audiam animal
+        molestiate te.
+      </Typography>
+    </>
+  );
+}
+
+function ContentDiplay(user) {
+  if (user === 'students') {
+    return (
+      <>
+        <LoginButton />
+        <Link to="/scholarships">
+          <Button
+            variant="outlined"
+            color="primary"
+            style={{ marginLeft: '20px' }}>
+            Find Scholarships
+          </Button>
+        </Link>
+      </>
+    );
+  }
+  return (
+    <>
+      <Link to="/home">
+        <Button variant="outlined" color="primary">
+          Submit Scholarship
+        </Button>
+      </Link>
+    </>
+  );
+}
```

Comment:
Few things, 

1. "ContentDiplay" is a poor name for what is just a list of buttons
2. Both cases are the same except that one of them has a `LoginButton`. This can be simplified to 9 lines if you use the ternary operator and `&&` to conditionally render the `LoginButton`. 
3. There's a way to envelop `<Link>` with other elements. I think it's something like
```jsx
      <Link to="/url">
        <Button buttonProperties>
          Button Name
        </Button>
      </Link>
```
can be written as
```jsx
    <Button component={Link} to="/url" buttonProperties>
      Button Name
    </Button>
```
I did something similar [here](https://github.com/beyondhb1079/s4us/blob/main/src/components/ScholarshipList.jsx#L64)

Reply:
Alright got it, reduced to 8 lines